packages/render-fidelity-tools/src/common.ts

```diff
@@ -26,6 +26,14 @@ export const MAX_COLOR_DISTANCE: number = 35215;
 
 export const DEVICE_PIXEL_RATIO: number = 2;
 
+// use this threshold to do automatic fidelity test for model-viewer. any
+// scenario whose rms value (in dB) is bigger than the threshold will fail.
+export const FIDELITY_TEST_THRESHOLD: number = -24;
+
+// use this threshold to detect whether the screenshot generate in fidlity test
+// is completely white or transparent
+export const WHITE_THRESHOLD: number = -10;
+
```

Comment:
I think we need to play back and force on these two thresholds.  The fidelity threshold is based on the highest value among all scenarios, the white threshold is based on the crazy metrics i showed you before (every renderers give around -9 dB)